index.html

```diff
@@ -0,0 +1,180 @@
+      <header>
+        <a href="#home" class="logo">
+          <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+        </a>
+        <nav>
+          <!-- mobile navigation starts here -->
+          <a href="#home" class="logo">
+            <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+          </a>
+          <div class="dropdown">
+            <div id="dropdownContent" class="dropdown-content">
+              <span class="close" onclick="closeDropdown()">&times;</span>
+              <a href="#home" onclick="closeDropdown()">Home</a>
+              <a href="./about.html" onclick="closeDropdown()">About</a>
+              <a href="#cards" onclick="closeDropdown()">Program</a>
+              <a href="#cards" onclick="closeDropdown()">Join</a>
+              <a href="#speakers" onclick="closeDropdown()">Speaker</a>
+            </div>
+          </div>
+          <div class="desk-ul">
+            <a href="#home" class="desk-li">News</a>
+            <a href="./about.html" class="desk-li">About</a>
+            <a href="#cards" class="desk-li">Program</a>
+            <a href="#cards" class="desk-li">Join</a>
+            <a href="#speakers" class="desk-li">Speaker</a>
+            <a href="#home" class="desk-li">ULD Compign</a>
+          </div>
+          <!-- mobile navigation endss here -->
+        </nav>
+      </header>
+      <!-- header starts here -->
+
+      <!-- home page starts here -->
+      <div id="home" class="home">
+        <div class="headline">
+          <h3>"Hello! Sharing World"</h3>
+          <h1>UNITING THE URBAN DESIGN AND LANDSCAPE EXPO 2023</h1>
+          <p class="headline-paragraph">
+            Urban Design & Landscape Expo is a prominent platform for the urban
+            design and landscape industry to expand your business in the Middle
+            East, Africa and South Asia. The event offers live product
+            showcases, knowledge exchange, and networking opportunities, all
+            alongside the prestigious Big 5 Global event.
+          </p>
+          <h2>2023.10.16(MON) ~ 19(THU)</h2>
+          <p class="foot-p">
+            @ DMG events is an organiser of face-to-face events and a publisher
+            of trade magazines for the international Construction, Energy,
+            Coatings, Transport, Hospitality & Design industries.
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- home page ends here -->
+    <!-- cards starts here -->
+    <section id="cards" class="cards">
+      <p>Main Program</p>
+      <hr class="about-hr" />
+      <div class="card-group">
+        <div class="card">
+          <img src="./assests/lecture.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Lecture</h5>
+            <p class="card-text">
+              Listen to the speakers from the various countries about the
+              messages of sharing and opening
```

Comment:
- [optional] i suggest you work on the more button such that when it is clicked it goes to the bottom of the speakers and change to see less. This will make it look presentable
![Screenshot (1201)_LI](https://github.com/Kaiserabbas/Capstone/assets/65298048/f4be4ce5-c47e-459e-899a-4036bee4369d)